packages/nimble-components/src/rich-text/viewer/styles.ts

```diff
@@ -48,6 +48,10 @@ export const styles = css`
         display: none;
     }
 
+    nimble-anchor {
+        font: unset;
```

Comment:
It's almost never worth using the value `unset`. Folks generally don't know how it behaves.
In this case it is equivalent to setting it to `font: inherit` since font is an inheritable property.
But I think overriding the entire font isn't the correct approach. We specifically want to let the font-weight and font-style through and let the anchor keep it's other font styling so we should just do those.

```suggestion
        font-weight: inherit;
        font-style: inherit;
```